test_demo_any.py

```diff
@@ -0,0 +1,27 @@
+from os import path as osp
+
+import numpy as np
+
+from model import DeepLab
+from utils import (Dataset, Iterator, save_load_means, subtract_channel_means, single_demo, read_image)
+
+if __name__ == '__main__':
+
+    demo_dir = 'data/demos/deeplab/MyImg/'
+    models_dir = 'data/models/deeplab/resnet_101_voc2012/'
+    model_filename = 'resnet_101_0.6959.ckpt'
+    image_filename='data/datasets/MyImg/JPEGImages/00.jpg'
+
+    channel_means = save_load_means(means_filename='channel_means1.npz',image_filenames=None, recalculate=False)
```

Comment:
Here we should use `channel_means.npz` instead of `channel_means1.npz` for compatibility with the current model settings.